packages/synapse-react-client/src/components/widgets/facet-nav/FacetNav.tsx

```diff
@@ -30,7 +34,7 @@ export type FacetNavProps = {
 }
 
 type UiFacetState = {
-  name: string
+  facet: UniqueFacetIdentifier
```

Comment:
Facets are now defined using the 2-tuple (columnName, jsonPath), so we can no longer track UI state using just a column name. Use the `UniqueFacetIdentifier` record that stores this data